src/index.html

```diff
@@ -7,49 +7,49 @@
     <header>
       <h1>Travel Tracker</h1>
     </header>
-    <div class="page" id="loginPage">
+    <div class="page loginPage" id="loginPage">
       <form id="login">
-        <input id="userName" type="text" autocomplete="off" placeholder="Enter User Name..." required />
-        <input id="userPassword" type="text" autocomplete="off" placeholder="Enter Password..." required />
-        <input id="loginSubmit" type="submit" value="Sign In" />
+        <input class="inputField loginInputField" id="userName" type="text" autocomplete="off" placeholder="Enter User Name..." required />
```

Comment:
You could probably add some ARIA tags for screen readers on these inputs for better accessibility score